app/src/main/java/org/thoughtcrime/securesms/registration/ui/countrycode/CountryCodeSelectScreen.kt

```diff
@@ -269,9 +269,10 @@ private fun SearchBar(
     shape = RoundedCornerShape(32.dp),
     modifier = modifier
       .background(MaterialTheme.colorScheme.background)
-      .padding(bottom = 18.dp, start = 16.dp, end = 16.dp)
+      .padding(bottom = 18.dp)
+      .padding(horizontal = 16.dp)
       .fillMaxWidth()
-      .height(54.dp)
+      .defaultMinSize(54.dp)
```

Comment:
we only need to set the `minHeight` to `54.dp`, and can leave the `minWidth` as `Dp.Unspecified`

```suggestion
      .defaultMinSize(minHeight = 54.dp)
```

Reply:
true indeed